### PR TITLE
Bookstore: initialize project with components

### PR DESCRIPTION
**Pull request by marurevi**
## Project Setup
- Initialise React app and create a directory for the components.
- Add React Router and set two Routes and Links for the app's navigation:
  - Books 

  - Categories

### CODE REVIEW

File: src/components/Book.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  const { topic, title, author } = props;
+  return (
+    <div className="container">
+      <ul>
+        <li>{topic}</li>
+        <li>{title}</li>
+        <li>{author}</li>
+      </ul>
+      <button type="button">UPDATE PROGRESS</button>
+    </div>
```
Comment (vic778) [inline src/components/Book.js]: - [x] Kindly this `div` should contains a remove button also as required in the requirement of the project `Should display the list of books (empty at this point but it should be ready for the data) with the Remove button (no funcionality yet). `
- [x] As required the style or any functionality for now is not require. Add the remove button to your project and let's see it approve :handshake: 

Your design
Screenshot from 2022-06-03 03-48-02

Project with remove b
Screenshot from 2022-06-03 03-43-14
utton

**Review by vic778 — CHANGES_REQUESTED** [review]
## Hello @marurevi 

Good job so far! I know you have worked hard on making this project.  There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
## Required Changes ♻️

- [x] Kindly check comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/AddNewBook.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        <select name="category">
+          <option value="" defaultValue>Category</option>
+          <option value="Adventure">Adventure</option>
+          <option value="Science Fiction">Science Fiction</option>
+          <option value="Terror">Terror</option>
+          <option value="Romance">Romance</option>
+          <option value="Economy">Economy</option>
+          <option value="Technology">Technology</option>
+        </select>
+        <button type="submit">ADD BOOK</button>
```
Comment (Kingstalux) [inline src/components/AddNewBook.js:20]: - [x] (OPTIONAL) This is not really optional. In the add book form, you are to have inputs for the title and author of a book and not the title and category. Kindly rectify this issue 👍 
- Requirements:
Annotation 2022-06-03 051845

**Review by Kingstalux — APPROVED** [review]
Hi @marurevi,
# Status: Approved ✔️ ✔️ ✔️ 
 
68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f78543078657a514755357843444a754350652f67697068792e676966


Good job implementing the required changes 👎 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

- [x] Kindly remove all the unnecessary scripts in your README file created from the `npx create-react-app` and instead put information on how a user can set up and run your project locally on their machine (`npm i, npm start`). Also kindly fix the broken image to make your README file more professional 👍 
- Your README:
Annotation 2022-06-03 052458


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**